routetable/testing_shims.go

```diff
@@ -17,60 +17,130 @@ package routetable
 import (
```

Comment:
I think it's a bit confusing for this file to be named so that it looks like it's part of our UT, when in fact - IIUC - it is very much product code.

Is this also related to your query about how you can test UT code?  If it isn't actually UT code, presumably that problem is no longer there...?